client/layout/html-is-iframe-classname.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { useEffect, memo } from 'react';
+import { useSelector } from 'react-redux';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import isIframeForHtmlElement from 'state/selectors/is-iframe-for-html-element';
+
+const debug = debugFactory( 'calypso:layout:html-is-iframe-classname' );
+
+const HtmlIsIframeClassname = memo( () => {
+	const isIframe = useSelector( isIframeForHtmlElement );
+
+	useEffect( () => {
+		const htmlNode = document.querySelector( 'html' );
+
+		if ( ! htmlNode ) {
+			debug( 'no html node' );
+			return;
+		}
+
+		if ( isIframe ) {
+			debug( 'adding is-iframe' );
+			htmlNode.classList.add( 'is-iframe' );
+		} else {
+			debug( 'removing is-iframe' );
+			htmlNode.classList.remove( 'is-iframe' );
+		}
+	} );
+
+	return null;
+} );
+
+export default typeof document === 'object' ? HtmlIsIframeClassname : () => null;
```

Comment:
You can just export `HtmlIsIframeClassname` without guarding for `document`. As the `document` is touched only in `useEffect`, the component is SSR-safe anyway. The Node.js server will never run the `useEffect` handler.